src/app/shared/form/builder/ds-dynamic-form-ui/relation-lookup-modal/relationship.effects.ts

```diff
@@ -161,11 +161,13 @@ export class RelationshipEffects {
   }
 
   private addRelationship(item1: Item, item2: Item, relationshipType: string, submissionId: string, nameVariant?: string) {
+    console.log('addRelationship 1');
```

Comment:
This looks to be accidental & can be removed

Reply:
Done